articles/service-fabric/how-to-managed-cluster-modify-node-type.md

```diff
@@ -298,6 +298,66 @@ To modify the OS image used for a node type using an ARM Template, adjust the `v
 }
 ```
 
+## Modify disc size on an existing node type
+Service Fabric Managed Clusters enables you to modify the disc size for new nodes in an exsisting node type in place.  As you deploy and run your application, load increases or decreases may cause a need to change the disc size on a node type.  
+
+[!NOTE]
+While Azure is making updates to the node type, other changes cannot be made to the node type.  Service Fabric Managed Clusters will spin up new nodes with the new disc size; therefore, the old nodes have to be scaled down.  See the below on procedure to scale a node type.
+
+### Modify disc size using portal
+In this walkthrough, you will learn how to modify a disc size using portal.
+
+1) Log in to [Azure portal](https://portal.azure.com/)
+
+2) Navigate to your cluster resource Overview page. 
+![Sample Overview page][overview]
+
+3) Select `Node Types` under the `Settings` section 
+
+4) Select `Change size` under the Data disk size
+
+5) Pick the size you wish and select ok
+
+6) select `Apply` at the bottom and wait for the changes to be applied
```

Comment:
What is the impact to the running workloads? Does it follow normal UD walk? Can we reference that "normal" procedure SFMC follows?

Reply:
+1, I actually was not sure so I did a search and got the following:

_Disk is the preferred spelling in American English, and it's also the spelling used for computer-related objects, such as a hard disk. Disc is the preferred spelling in British English, and it's also the spelling used for devices carrying sound._